lib/index.js

```diff
@@ -61,18 +67,19 @@ function TradeOfferManager(options) {
 	}
 
 	if (this._steam) {
-		this._steam.on('tradeOffers', (count) => {
+		this._steam.on('tradeOffers', () => {
```

Comment:
@DoctorMcKay I was thinking about it too. Probably it'd be better to disable ESLint for this lines.